flagparse/flags.go

```diff
@@ -32,49 +32,36 @@ import (
 const (
 	ConfigFile = "configFile"
 
-	Files        = "files"
-	DefaultFiles = ""
+	Files = "files"
 )
 
 //UploadFileConfig describes file config of uploadable
 type UploadFileConfig struct {
-	*client.BrokerConfig
-	*client.UploadableConfig
-	*logger.LogConfig
+	client.BrokerConfig
+	client.UploadableConfig
+	logger.LogConfig
 
-	Files string `json:"files,omitempty"`
+	Files string `json:"files,omitempty" descr:"Glob pattern for the files to upload"`
+}
+
+//ConfigNames contains template names to be replaced in config properties descriptions and default values
+var ConfigNames = map[string]string{
+	"name": "Autouploadable", "feature": "Uploadable", "period": "Upload period",
+	"action": "upload", "actions": "uploads", "running_actions": "uploads",
```

Comment:
Do we need actions and running_actions when the values are the same?

Reply:
Yes, it will be used for backup, where actions are different.